src/app/state/accountHistorySlice.ts

```diff
@@ -115,6 +115,9 @@ export const accountHistorySlice = createSlice({
     setSelectedTable: (state, action: PayloadAction<Tables>) => {
       state.selectedTable = action.payload;
     },
+    resetOrderHistory: (state) => {
+      state.orderHistory = [];
```

Comment:
Can you remove the trades as well?
```suggestion
      state.orderHistory = [];
      state.trades = [];
```